### PR TITLE
Create folders per Grade + README example

### PR DESCRIPTION
**Pull request by bodacd**

### CODE REVIEW

**Review by diogotia — APPROVED** [review]

**Review by endalexei — APPROVED** [review]

File: JT/Example of README.md [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+<details>
+<summary>Solution</summary>
+
+- install **jest-environment-jsdom**:
+```shell
+npm install -D jest-environment-jsdom
+```
+</details>
+
+### B". Error: Document is not defined
```
Comment (StazMD) [inline JT/Example of README.md:44]: Duplicating section B.

File: JT/Example of README.md [hunk 2]
```diff
@@ -0,0 +1,411 @@
+# Front-End Unit Test Workshop - Fixers
+
+### A. Error: Jest encountered an unexpected token
+Configure jest to compile ts files.
+
+<details>
+<summary>Solution</summary>
+
+- install **babel/preset-env**:
+- install **babel/preset-react**
```
Comment (StazMD) [inline JT/Example of README.md:10]: Missing colon at the end of the line. Would be nice to add everywhere where needed.

File: JT/Example of README.md [hunk 3]
```diff
@@ -0,0 +1,411 @@
...
+<summary>Solution</summary>
+
+- Mock `matchMedia` in `SetupJest.js` file
+```js
+import {jest} from '@jest/globals';
+import '@testing-library/jest-dom';
+import React from 'react';
+
+global.React = React;
+global.ResizeObserver = require('resize-observer-polyfill')
```
[the remark this answers is not in the document]
Comment (diogotia) [reply JT/Example of README.md:395]: Just a note : In JavaScript, semicolons are used to end a statement. While they are not always required.

File: JT/Example of README.md (hunk 1 above)
Comment (bodacd) [reply JT/Example of README.md:44]: ther's two B in  README example, with `'` sign to define first and second step in this way - B' and B''

File: JT/Example of README.md (hunk 2 above)
Comment (bodacd) [reply JT/Example of README.md:10]: this README file have purpose to show an example of how we can use markdown. At the moment we can skip the content.

**Review by endalexei — APPROVED** [review]

**Review by StazMD — APPROVED** [review]

**Pull request merged**